corehq/apps/callcenter/sync_usercase.py

```diff
@@ -176,12 +176,12 @@ def _not_same(val1, val2):
     return changed_fields
 
 
-def get_sync_lock_key(user_id):
-    return ["sync_user_case_for_%s" % user_id]
+def get_sync_lock_key(user_id, domain):
+    return [f"sync_user_case_for_{user_id}_{domain}"]
 
 
-def sync_call_center_user_case(user):
-    with CriticalSection(get_sync_lock_key(user._id)):
+def sync_call_center_user_case(user, domain):
```

Comment:
Not blocking: It looks like this is only used in tests. If so, it can be deleted.